en/starter/generator/core_concepts.md

```diff
@@ -239,10 +239,6 @@ This service uses the [SwiftMetrics package](https://github.com/RuntimeTools/Swi
 
 The connection details for this client are loaded by the configuration boilerplate code and are passed to the SwiftMetrics auto-scaling client.
 
-### Non-IBM Cloud services
-
-Non-IBM Cloud services can be hosted anywhere including your local machine.
-

```

Comment:
You didn't remove the whole section -- local CouchDB and Redis are not supported any more, they need to be removed also.

Reply:
done